modules/renter/contractor/uptime_test.go

```diff
@@ -0,0 +1,131 @@
+package contractor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// uptimeHostDB overrides an existing hostDB so that it always returns
+// isOffline == true for a specified address.
+type uptimeHostDB struct {
+	hostDB
+	addr modules.NetAddress
+}
+
+func (u uptimeHostDB) Host(addr modules.NetAddress) (modules.HostDBEntry, bool) {
```

Comment:
This should have a docstring. Repeating yourself in comments I feel is okay because it makes it easier to navigate - otherwise the reader has to hunt around more to figure out what a function does.

In this case it's nontrivial relative to the name. It doesn't just return a Host, it returns an offline host. That's definitely significant enough to merit a docstring.